go/noise/gaussian_noise_test.go

```diff
@@ -856,6 +896,30 @@ func TestComputeConfidenceIntervalFloat64Gaussian(t *testing.T) {
 			want:            ConfidenceInterval{-96.6140883158, 236.6140883158},
 			wantErr:         false,
 		},
+		// float64 precision should be accurate for abs(noisedX) < 2^54
+		{
+			desc:            "Large positive noisedX",
+			noisedX:         38475693,
```

Comment:
IIUC, this is actually a float value. It makes your code easier to read and understand if you signify integral float values by writing them as 38475693.0.

Please also fix in other places.